components/inputs/documentation/date-time-inputs.md

```diff
@@ -0,0 +1,109 @@
+| `label-hidden` | Use if label should be visually hidden but available for screen reader users |
+
+**Events:**
+
+* `d2l-input-date-change`: dispatched when a date is selected or typed. `value` reflects the selected value and is in ISO 8601 calendar date format (`YYYY-MM-DD`).
+
+## Time Inputs
+
+The `<d2l-input-time>` component consists of a text input field for typing a time and an attached dropdown for time selection. The dropdown opens on click of the text input, or on enter or down arrow press if the text input is focused. It displays the `value` if one is specified, or a placeholder if not, and reflects the selected value when one is selected in the dropdown or entered in the text input.
+
+![example screenshot of time input](../screenshots/time.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-time.js';
+</script>
+<d2l-input-time
+  label="Start Time"
+  value="18:00:00">
+</d2l-input-time>
+```
+
+**Properties:**
+
+- `label` (String, required): accessible label for the input
+- `disabled` (Boolean): disables the input
+- `enforce-time-intervals` (Boolean): rounds up to nearest valid interval time (specified with `time-interval`) when user types a time
+- `label-hidden` (Boolean): hides the label visually (moves it to the input's `aria-label` attribute)
+- `time-interval` (String, default: `thirty`): number of minutes between times shown in dropdown. Valid values include `five`, `ten`, `fifteen`, `twenty`, `thirty`, and `sixty`.
+- `value` (String, default: `''`): value of the input. This should be in ISO 8601 time format (`hh:mm:ss`) and should be localized to the user's timezone (if applicable).
+
+**Accessibility:**
+
+To make your usage of `d2l-input-time` accessible, use the following properties when applicable:
+
+| Attribute | Description |
+|--|--|
+| `label` | **REQUIRED** [Acts as a primary label on the input](https://www.w3.org/WAI/tutorials/forms/labels/). Visible unless `label-hidden` is also used. |
+| `label-hidden` | Use if label should be visually hidden but available for screen reader users |
+
+**Events:**
+
+* `change`: dispatched when a time is selected or typed. `value` reflects the selected value and is in ISO 8601 time format (`hh:mm:ss`).
+
+## Date-Time Inputs
+
+The `<d2l-input-date-time>` component consists of a `<d2l-input-date>` and a `<d2l-input-time>` component. The time input only appears once a date is selected. This component displays the `value` if one is specified, and reflects the selected value when one is selected or entered.
+
+![example screenshot of date input](../screenshots/date-time.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-date-time.js';
+</script>
+<d2l-input-date-time
+  label="Start Date"
+  value="2020-11-20T12:00:00.000Z">
+</d2l-input-date-time>
+```
+
+**Properties:**
+
+- `label` (String, required): accessible label for the input
```

Comment:
Did we decide against supporting `label-hidden` here? I'm fine with that, just curious if it was on purpose. Can always add later.

Reply:
`input-date-time` uses a fieldset which as far as I can tell does not support a hidden label. That could be implemented if it's something we want/that is okay to implement on a fieldset.